packages/migrations/src/Component/Doctrine/Migrations/Configuration.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace Shopsys\MigrationBundle\Component\Doctrine\Migrations;
+
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Migrations\Configuration\Configuration as DoctrineConfiguration;
+use Doctrine\DBAL\Migrations\Finder\MigrationFinderInterface;
+use Doctrine\DBAL\Migrations\OutputWriter;
+use Doctrine\DBAL\Migrations\QueryWriter;
+use Doctrine\DBAL\Migrations\Version;
+
+class Configuration extends DoctrineConfiguration
```

Comment:
This looks to me like duplication of the ordering logic... Who's responsibility is it to order the migrations? Finder's or Configuration's?

Reply:
Well, finders find migrations, but it was not supposed in multiple directories, so it orders them default by timestamp. Then Configuration orders migrations using MigrationsConfig the way it is supposed by setting in migrations_config.yml